packages/common/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@biconomy/common",
+  "version": "3.1.2",
```

Comment:
shouldn't this version be same as other package versions?

Reply:
ok just saw we can update them all together to 3.1.3 or 4.0.0 for now. 
because common 3.1.2 already exists in npm registry